src/wp-includes/block-supports/elements.php

```diff
@@ -31,90 +31,77 @@ function wp_get_elements_class_name( $block ) {
  * @return string Filtered block content.
  */
 function wp_render_elements_support( $block_content, $block ) {
-	if ( ! $block_content || empty( $block['attrs'] ) ) {
+	if ( ! $block_content || ! isset( $block['attrs']['style']['elements'] ) ) {
 		return $block_content;
 	}
 
-	$block_type = WP_Block_Type_Registry::get_instance()->get_registered( $block['blockName'] );
+	$block_type       = WP_Block_Type_Registry::get_instance()->get_registered( $block['blockName'] );
+	$style_attributes = $block['attrs']['style']['elements'];
 
```

Comment:
```suggestion
  if( ! $block_type ){
     return $block_content;
  }
```

Should we do this as well?